applications/TrilinosApplication/tests/test_trilinos_linear_solvers.py

```diff
@@ -296,6 +296,7 @@ def test_amesos2_basker_2(self):
             """)
 
 class TestAztecLinearSolvers(TestLinearSolvers):
+    @KratosUnittest.skipIf(not hasattr(KratosMultiphysics.TrilinosApplication, 'AztecSolver'), "AztecSolver was explicitly disabled.")

```

Comment:
```suggestion
@KratosUnittest.skipUnless(hasattr(KratosMultiphysics.TrilinosApplication, 'AztecSolver'), "AztecSolver was explicitly disabled.")
class TestAztecLinearSolvers(TestLinearSolvers):

```
same can be done for the other tests, the Aztec and the ML (unlike the Amesos) have no options to configure so you can disable the entire test class

Reply:
nice, done